loris/img_info.py

```diff
@@ -372,8 +372,7 @@ def get(self, request):
                 info_and_lastmod = (info, lastmod)
                 logger.debug('Info for %s read from file system', request)
                 # into mem:
-                self._dict[request.url] = info_and_lastmod
-
+                self.__setitem__(request, info)
```

Comment:
__setitem__ writes the content to the file system as well as memory, doesn't it? Don't we only want this information to go into memory in this get method?

Maybe we should split __setitem__ up into two methods, one for writing to disk and one for putting in memory?

Reply:
Hmm, I see your reasoning here. Writing on every ```get()``` would be a lot of writing to disk.

In this case I would be more inclined to tack a kwarg into ```__setitem__``` for this purpose. The ```__setitem__``` and ```__getitem__``` dict-style interface is what is meant to be presented to "the outside world" from this class, correct?

If that is the case (and the cache is meant to be swappable in the wider code base by purpose-built implementations which expose the same interface), I would think that this case would be  the only one where bootstrapping something into RAM that already exists on the filesystem would be required - all other cases would require being written both to disk and into RAM, both within the class and in calling code.